frontends/web/src/api/coins.ts

```diff
@@ -46,3 +46,16 @@ export type TAmount = {
 export const parseExternalBtcAmount = (amount: string): Promise<TAmount> => {
   return apiGet(`coins/btc/parse-external-amount?amount=${amount}`);
 };
+
+export type AddressSignResponse = {
+  success: boolean;
+  signature: string;
+  address: string;
+  errorMessage?: string;
+  errorCode?: BackendErrorCode;
+}
+
+export const signAddress = (format: string, msg: string, accountCode: AccountCode): Promise<AddressSignResponse> => {
+  return apiPost('coins/btc/sign-address', { format, msg, accountCode });
```

Comment:
i wonder why the sign function and endpoints are not added to btc/account.go and btc/handlers.go, since it's all BTC specific.

Reply:
Good point, done.